src/components/Author/styles.module.scss

```diff
@@ -0,0 +1,58 @@
+@import 'scss/variables/_colors';
+
+.container {
+  position: relative;
+
+  .container-author {
+    align-items: center;
+    display: flex;
+    flex-direction: row;
```

Comment:
You could use the utility classes that come in the bootstrap to handle flex properties.